homeassistant/components/tuya/number.py

```diff
@@ -234,6 +234,14 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Smart Gardening system
+    # https://developer.tuya.com/en/docs/iot/categorysz?id=Kaiuz4e6h7up0
+    "sz": (
+        NumberEntityDescription(
+            key=DPCode.BRIGHT_VALUE,
+            translation_key="brightness",
+        ),
+    ),
```

Comment:
This should be part of the light, right?

Reply:
Moved and merged with desc `DPCode.LIGHT`